package com.kryszak.gwatlin.clients.exception

internal data class RetrieveError(val text: String)